app/boards/shields/knob_goblin/Kconfig.defconfig

```diff
@@ -0,0 +1,45 @@
+if SHIELD_KNOB_GOBLIN
+
+config ZMK_KEYBOARD_NAME
+  default "Knob Goblin"
+
+config ZMK_USB
+  default y
+
```

Comment:
You don't need to add this configuration. The board that is plugged in will dictate whether this is needed.
```suggestion
```